worker/include/RTC/RTCP/FeedbackPsFir.hpp

```diff
@@ -27,7 +27,12 @@ namespace RTC
 			{
 				uint32_t ssrc;
 				uint8_t sequenceNumber;
+#ifdef _MSC_VER
+				// alignment
+				uint8_t reserved[3];
```

Comment:
Why is this required in Windows? (just wondering). And, if so, is alignment just required in this file?

BTW: Please always capitalize comments, add a dot at the end, and put them in the same line if possible:

```c++
uint8_t reserved[3]; // Alignment.
```

Reply:
visual studio seems do not support alignment like this `uint32_t reserved : 24;`, so just use `uint8_t reserved[3];` here.